src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
Here, your random instance is created every time the method is called. random - is our dependency, let's initialize dependencies using a constructor or directly create a new instance in the class fields.